sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -274,6 +274,8 @@ RUN apt-get update && apt-get install -y \
         python3-pytest-cov \
         python3-parse \
 # For snmpd
+        libnetsnmptrapd40=5.9+dfsg-3+b1 \
```

Comment:
Why we need to specify the version? Can we install version 5.9+dfsg-4+deb11u1?
If really want to downgrade the version, we can install it in the next docker build RUN step.

Reply:
> Hi Shilong, the build issue should be fixed with #11763.

@liushilongbuaa , maybe we can close the PR.